src/resolve-dependency.js

```diff
@@ -90,11 +90,14 @@ function getExportsTarget (exports, conditions, cjsResolve) {
           condition === 'import' && !cjsResolve ||
           conditions.includes(condition)) {
         const target = getExportsTarget(exports[condition], conditions, cjsResolve);
-        if (target !== undefined)
+        if (target !== undefined && (target === null || target.startsWith('./')))
```

Comment:
Is it expected that this `if` statement does not match the one on line 82 above?

Reply:
Yes they should be the same - fixed.

Reply:
@guybedford They are a little different still. Why use `continue` with one and not the other?

Reply:
Right, they are written differently but do the same thing. So the definition of the array fallback is that it will continue on invalid items, which is supposed to be different from the object conditions form. But I just realise now we're not actually doing that either! Done.

Note the extreme exports edge cases get pretty complex but aren't needed in 99% of cases... so we are just rattling out the `"exports": [{}, { "asdf": null }, null]` kinds of odd behaviours here.

Reply:
@guybedford I don't see a test for the array of `exports` like you mentioned above? They are all object `exports` as far as I see. Can we add a test for array?

Reply:
Yes there is a test at https://github.com/vercel/node-file-trace/pull/129/files#diff-fb129c0544238468b2fbaf801b451a7f for the fallback support.